src/user_workspaces_server/controllers/resources/slurm_api_resource.py

```diff
@@ -103,7 +103,7 @@ def launch_job(self, job, workspace, resource_options):
         except Exception:
             logger.info(slurm_response.text)
             raise APIException(
-                f"Slurm response for {job.id} could not be deciphered: {slurm_response.text}"
+                f"Slurm response for {job.job_details['id']} could not be deciphered: {slurm_response.text}"
```

Comment:
Verify

Reply:
I believe `job` here is an instance of a subclass of AbstractJob (e.g. JupyterLabJob) so if we want the ID of the database-row job that it represents, that would be in `job.job_details`. @jpuerto-psc

Reply:
That makes sense! ty for the fix.